deploy/crds/openebs_v1alpha1_openebsinstalltemplate_cr.yaml

```diff
@@ -1,146 +1,153 @@
+    image: quay.io/openebs/provisioner-localpv
+    imageTag: 2.11.1
+    replicas: '1'
     basePath: "/var/openebs/local"
-    nodeSelector: {}
-    tolerations: []
-    affinity: {}
+    enableLeaderElection: true
     healthCheck:
-      initialDelaySeconds: 30
-      periodSeconds: 60
-  
+      initialDelaySeconds: '30'
+      periodSeconds: '60'
   snapshotOperator:
+    enabled: true
     controller:
-      image: "quay.io/openebs/snapshot-controller"
-      imageTag: "2.10.0"
+      image: quay.io/openebs/snapshot-controller
+      imageTag: 2.11.0
     provisioner:
-      image: "quay.io/openebs/snapshot-provisioner"
-      imageTag: "2.10.0"
+      image: quay.io/openebs/snapshot-provisioner
+      imageTag: 2.11.0
     replicas: 1
     enableLeaderElection: true
-    upgradeStrategy: "Recreate"
-    nodeSelector: {}
-    tolerations: []
-    affinity: {}
+    upgradeStrategy: Recreate
     healthCheck:
       initialDelaySeconds: 30
       periodSeconds: 60
-  
   ndm:
-    image: "quay.io/openebs/node-disk-manager"
-    imageTag: "1.5.0"
+    enabled: true
+    image: quay.io/openebs/node-disk-manager
+    imageTag: 1.6.0
     sparse:
       path: "/var/openebs/sparse"
-      size: "10737418240"
-      count: "0"
+      size: '10737418240'
+      count: '0'
     filters:
       enableOsDiskExcludeFilter: true
       osDiskExcludePaths: "/,/etc/hosts,/boot"
-      enableVendorFilter: true
-      excludeVendors: "CLOUDBYT,OpenEBS"
-      includePaths: ""
+      enableVendorFilter: 'true'
+      excludeVendors: CLOUDBYT,OpenEBS
       enablePathFilter: true
+      includePaths: ''
       excludePaths: "/dev/loop,/dev/fd0,/dev/sr0,/dev/ram,/dev/dm-,/dev/md,/dev/rbd,/dev/zd"
-    nodeSelector: {}
     probes:
       enableSeachest: false
     healthCheck:
       initialDelaySeconds: 30
       periodSeconds: 60
-
   ndmOperator:
-    image: "quay.io/openebs/node-disk-operator"
-    imageTag: "1.5.0"
+    image: quay.io/openebs/node-disk-operator
+    imageTag: 1.6.0
     replicas: 1
     upgradeStrategy: Recreate
-    nodeSelector: {}
-    tolerations: []
     healthCheck:
-      initialDelaySeconds: 30
-      periodSeconds: 60
+      initialDelaySeconds: 15
+      periodSeconds: 20
     readinessCheck:
-      initialDelaySeconds: 4
-      periodSeconds: 10
-      failureThreshold: 1
-    cleanupImage: "quay.io/openebs/linux-utils"
-    cleanupImageTag: "2.10.0"
-
-
+      initialDelaySeconds: 5
+      periodSeconds: 20
   webhook:
     enabled: true
-    image: "quay.io/openebs/admission-server"
-    imageTag: "2.10.0"
-    failurePolicy: "Fail"
-    generateTLS: true
+    image: quay.io/openebs/admission-server
+    imageTag: 2.11.0
+    failurePolicy: Fail
     replicas: 1
     healthCheck:
       initialDelaySeconds: 30
       periodSeconds: 60
-    nodeSelector: {}
-    tolerations: []
-    affinity: {}
     hostNetwork: false
-
+  helper:
+    image: quay.io/openebs/linux-utils
+    helperImageTag: 2.11.0
+  featureGates:
```

Comment:
Is this part of NDM config?

Reply:
@kmova I checked it from https://github.com/openebs/charts/blob/9beb1dd261939fe22d758bbfa9b0b1814d6d3d87/charts/openebs/values.yaml#L250 the helper and feature gates are not under ndm operator but at same level